src/qamqpframe.cpp

```diff
@@ -58,7 +73,7 @@ QDataStream &operator<<(QDataStream &stream, const QAmqpFrame &frame)
 
     // write end
     stream << qint8(QAmqpFrame::FRAME_END);
-    stream.device()->waitForBytesWritten(1000);
+    stream.device()->waitForBytesWritten(frame.writeTimeout());
```

Comment:
yes? Do you prefer me to call it with the static form? (even if it's the same result)